docs/source/release_notes/release-notes.rst

```diff
@@ -50,6 +50,10 @@ develop
     *    - Type
          - Change
 
+    *    - |improved|
```

Comment:
I don't think a release note is needed here - nobody will be impacted apart from Atlas Devs.

Reply:
DIscussed offline - it seems there may be some people (positively!) impacted, so I'm ok with keeping it in.